tests/test_core.py

```diff
@@ -602,19 +618,19 @@ def test_truediv():
         ndx.nint16,
         ndx.nint32,
         ndx.nint64,
-        ndx.uint8,
```

Comment:
Does the test fail for these unsigned data types? If so, maybe we could keep the test but mark it as xfail?

Reply:
These fail because of lack of kernel support and there is the test for this here: https://github.com/Quantco/ndonnx/pull/56/files#diff-938de46e643ab091cff6a7c23e4e752e8907e2266f47889d988923352f7a1058R636-R650. We previously did not follow the correct dtype determination algorithm but we could also just make the default unsigned int type uint32.

Reply:
I changed my mind about creating a breaking change in https://github.com/Quantco/ndonnx/pull/58.